pandas/_libs/groupby_helper.pxi.in

```diff
@@ -106,11 +108,20 @@ def group_last(rank_t[:, :] out,
                     if nobs[i, j] == 0:
                         if rank_t is int64_t:
                             out[i, j] = NPY_NAT
+                        elif rank_t is uint64_t:
+                            runtime_error = True
+                            break
                         else:
                             out[i, j] = NAN
                     else:
                         out[i, j] = resx[i, j]
 
+    if runtime_error:
+        # We cannot raise directly above because that is within a nogil
```

Comment:
where are these caught?

Reply:
this would get caught by one of the `except Exception`s in the groupby code that i'm trying to make more specific.